dependencies.gradle

```diff
@@ -15,7 +15,7 @@ ext {
     jackson_version = '2.9.6' // Keep this version for compatibility with Android 4.4 (API level 19)
     json_version = '20180813'
     junit_version = '4.12'
-    kotlin_version = '1.3.41'
+    kotlin_version = '1.3.50'
```

Comment:
I get this warning:

```Kotlin version that is used to build with Gradle (1.3.50) differs from the one bundled into the IDE plugin (1.3.41)```

Should we go back to 1.3.41 ? Or wait for an Android Studio version that supports 1.3.50 ?